### ISSUE TITLE
FastQC plots disabled completely when configuring table_columns_visible property

### ISSUE THREAD

**Issue by mahesh-panchal** [ewels/MultiQC#1170]
**Description of bug:**
FastQC plots are disabled when I set `FastQC: false` in the `table_columns_visible` of the config.
My understanding from the [documentation](https://multiqc.info/docs/#hiding-columns) was that this only affected the general statistics table.

config snippet:
```
table_columns_visible:
    FastQC: false
    Flash:
        combopairs: true
    QualiMap:
        avg_gc: false
        1_x_pc: true
        median_coverage: true
```

**MultiQC run details (please complete the following):**
 - Command used to run MultiQC: `multiqc -c $config .` inside nextflow script.
 - MultiQC Version: MultiQC v1.8
 - Operating System: Linux
 - Python Version: Python 3.8.1
 - Method of MultiQC installation: docker image `quay.io/biocontainers/multiqc:1.8--py_2`

 **Additional context**

**Comment by mahesh-panchal** [comment]
Whoops. Duplicate of #1149

**Comment by ewels** [comment]
Hi @mahesh-panchal,

I'm not sure that this is a duplicate - you're talking about FastQC _plots_ here? #1149 is about bulk-hiding all table columns from a given module.

Phil

**Comment by ewels** [comment]
(Leaving the issue closed as I cannot replicate this with v1.9dev - the FastQC table columns are hidden as expected, but that's all).
